api/DashboardApi.ts

```diff
@@ -21,7 +21,7 @@ export interface IDashboardApi extends basem.ClientApiBase {
     createDashboard(dashboard: DashboardInterfaces.Dashboard, teamContext: TfsCoreInterfaces.TeamContext): Promise<DashboardInterfaces.Dashboard>;
     deleteDashboard(teamContext: TfsCoreInterfaces.TeamContext, dashboardId: string): Promise<void>;
     getDashboard(teamContext: TfsCoreInterfaces.TeamContext, dashboardId: string): Promise<DashboardInterfaces.Dashboard>;
-    getDashboards(teamContext: TfsCoreInterfaces.TeamContext): Promise<DashboardInterfaces.DashboardGroup>;

```

Comment:
Breaking change so good to bump major.

Reply:
Good catch - did update major, but was questioning that decision, but you're definitely right